src/chrome/content/rules/ESEA.net.xml

```diff
@@ -0,0 +1,17 @@
+<!--
+	Redirects:
+		- esea.net → play.esea.net
+		- news.esea.net → play.esea.net
+-->
+<ruleset name="ESEA.net">
+	<target host="esea.net" />
+	<target host="www.esea.net" />
+	<target host="fragshack.esea.net" />
+	<target host="mdl.esea.net" />
+	<target host="news.esea.net" />
+	<target host="play.esea.net" />
```

Comment:
+ `static.esea.net` (please also add a valid test url for it)

Reply:
@J0WI Done.